test/typing/plain_files/orm/column_expression_argument.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import ColumnElement
```

Comment:
**Federico Caselli** (CaselIT) wrote:

it would be nice to consolidate these into another existing file since the setup does not seems particularly complex

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5150

Reply:
I've moved it to `test/typing/plain_files/sql/common_sql_element.py`. Let me know if it's ok

Reply:
thanks, let's re-run the pipeline